cucumber-tests-platform/src/test/java/com/alliander/osgp/platform/cucumber/core/Helpers.java

```
@@ -26,309 +26,355 @@
+	}
+
+	/**
+	 *
+	 * @param settings
+	 * @param key
+	 * @param enumType
+	 * @param defaultValue
+	 * @return
+	 */
+	public static <E extends Enum<E>> E getEnum(final Map<String, String> settings, final String key,
+			final Class<E> enumType, final E defaultValue) {
+		if (!settings.containsKey(key)) {
+			return defaultValue;
+		}
+
+		return getEnum(settings, key, enumType);
+	}
+
+	public static <E extends Enum<E>> E getEnum(final Map<String, String> settings, final String key,
+			final Class<E> enumType) {
+		if (settings.get(key).isEmpty()) {
+			return null;
+		}
+		
+		return Enum.valueOf(enumType, settings.get(key));
+	}
+
+	/**
+	 * Check the correlationUid in the response and save it in the current
+	 * scenarioContext.
+	 *
+	 * @param response
+	 *            The response to find the correlationUid in.
+	 * @param organizationIdentification
+	 *            The organizationIdentifier used. Default test-org will be
+	 *            used.
+	 * @throws Throwable
+	 */
+	public static void saveCorrelationUidInScenarioContext(final String correlationUid,
+			String organizationIdentification) throws Throwable {
+		if (organizationIdentification == null || organizationIdentification.isEmpty()) {
+			organizationIdentification = Defaults.DEFAULT_ORGANIZATION_IDENTIFICATION;
+		}
+
+		// Validate the correlation-id starts with correct organization
+		Assert.isTrue(correlationUid.startsWith(organizationIdentification));
+		ScenarioContext.Current().put(Keys.KEY_CORRELATION_UID, correlationUid);
+	}
+
+	/**
+	 * When running automatic tests, it might be that not each project is
+	 * started in tomcat. When the repo's are cleared at the beginning of a test
+	 * run, you get some exceptions when the database wasn't found. Therefore
+	 * this method is created to ignore that.
+	 *
+	 * @param repo
+	 *            The repository to remove.
+	 */
+	public static <T extends AbstractEntity> void cleanRepoAbstractEntity(final JpaRepository<T, Long> repo) {
```

I still believe the "on error resume next" setup is a terribly bad code smell. If something fails, it had better fail fast and hard.

I understand the issue on local development machines, but maybe this is just a sign that we should not have placed all kinds of tests relating to different domains of the platform in a single project.